CMakePresets.json

```diff
@@ -14,16 +14,14 @@
       "name": "_debug-base",
       "hidden": true,
       "cacheVariables": {
-        "CMAKE_BUILD_TYPE": "Debug",
-        "CMAKE_CXX_FLAGS": "-fsanitize=address -fsanitize=pointer-compare -fsanitize=pointer-subtract -fsanitize=leak -fsanitize=undefined"
+        "CMAKE_BUILD_TYPE": "Debug"
```

Comment:
Is there a target that runs a sanitized build in the presets?

Reply:
No, but this preset does not work on APPLE!

Reply:
And attaching the sanitizer to the Debug build misses many of the things it is supposed to catch, in any case. And, in addition, specializing for the toolchain is what toolchain files are for, not the Presets.

OK. I wish it were otherwise, but I understand why the change is being requested.